packages/runtime/src/subscriptions.ts

```diff
@@ -0,0 +1,17 @@
+export let fetchSubscriptions : () => void = () => undefined;
+
+let fetchSubscriptionsPromise = new Promise<void>((resolve) => {
+  fetchSubscriptions = () => resolve();
+});
+
+export const stopFetchingSubscriptionsForTesting = () => {
```

Comment:
Used only in unit tests to reset the subscription state before the start of the next test. Not exported to the user.